### ISSUE TITLE
Load a csv file with multiple label columns

### ISSUE THREAD

**Issue by qingshan1975** [deeplearning4j/deeplearning4j#943]
Now I use 

                RecordReader recordReader = new CSVRecordReader(0,",");
                recordReader.initialize(new FileSplit(new ClassPathResource("train1000deps02.csv").getFile()));
                DataSetIterator iter = new RecordReaderDataSetIterator(recordReader,batchSize, 0, 2);

to load the csv file into an iterator. However, when there are multiple label columns, for example when the output is a vector,  RecordReaderDataSetIterator can not load the training data. Could you please support multiple label columns loading for csv files?

Thanks,

**Comment by AlexDBlack** [comment]
https://github.com/deeplearning4j/deeplearning4j/pull/1333

**Comment by lyphknoten** [comment]
I am trying to do the same. I found out, we can have more than 1 label if we use regression, but I tried so and it doesn't work as I want a pure classification. I have 1 label for each input (numInput=numOutput=numClassificationTasks, numClasses=5).
Any Suggestions?
